tests/test_submain.py

```diff
@@ -291,7 +315,10 @@ def get_tests(self, tier, type):
 
         self.last_tier = tier
 
-        for x in range(1,10): # Ten times because we care
+        for x in range(1, 10):  # Ten times because we care
+            # ^ Very witty. However, it would be nice to actually know
+            # exactly why we're yielding these ten times.
+
```

Comment:
Just to pretend there's many tests? To slow down the test suite? I have no idea, but this is a code smell. I'd try just removing the loop, and see if it breaks anything. If it does, maybe it'll give us a better idea of what this is for?